README.md

```diff
@@ -1,36 +1,446 @@
+     token = "$TERRAFORM_CLOUD_API_TOKEN"
+   }
+   ```
+
+1. Create a Kubernetes secret named `terraformrc` in the namespace.
+   Reference the credentials file (`$FILENAME`) created in the previous step.
+   ```shell
+   $ kubectl create -n $NAMESPACE secret generic terraformrc --from-file=credentials=$FILENAME
+   ```
+   Ensure `terraformrc` is the name of the secret, as it is the default secret name defined under the Helm value `syncWorkspace.terraformRC secretName` in the `values.yaml` file.
+
+If you have the free tier of Terraform Cloud, you will only be able to generate a token for the one team associated with your account. If you have a paid tier of Terraform Cloud, create a separate team for the `operator` with "Manage Workspaces" access.
+
+Note that a Terraform Cloud Team API token is a broad-spectrum token. It allows the token holder to create workspaces and execute Terraform runs. You cannot limit the access it provides to a single workspace or role within a team. In order to support a first-class Kubernetes experience, security and access control to this token must be enforced by [Kubernetes Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) policies.
+
+### Workspace Sensitive Variables
+
+Sensitive variables in Terraform Cloud workspaces often take the form of credentials for cloud providers or API endpoints. They enable Terraform Cloud to authenticate against a provider and apply changes to infrastructure.
+
+Create the secret for the namespace that contains all of the sensitive variables required for the workspace.
+
+```shell
+$ kubectl create -n $NAMESPACE secret generic workspacesecrets --from-literal=SECRET_KEY=$SECRET_KEY --from-literal=SECRET_KEY_2=$SECRET_KEY_2 ...
+```
+Ensure `workspacesecrets` is the name of the secret, as it is the default secret name defined under the Helm value `syncWorkspace.sensitiveVariables.secretName` in the `values.yaml` file.
+
+In order to support a first-class Kubernetes experience, security and access control to these secrets must be enforced by [Kubernetes Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) policies.
+
+### Terraform Version
+
+By default, the operator will create a Terraform Cloud workspace with
+[a pinned version](https://github.com/hashicorp/terraform-k8s/blob/master/operator/version/version.go) of Terraform.
+
+Override the Terraform version that will be set for the workspace by changing the Helm value `syncWorkspace.terraformVersion` to the Terraform version of choice.
+
+## Deploy the Operator
+
+Use the [Helm chart](https://github.com/hashicorp/terraform-helm) repository to deploy the Terraform Operator to the namespace you previously created.
+
+```shell
+$ helm repo add hashicorp https://helm.releases.hashicorp.com
+$ helm search repo hashicorp/terraform --devel
+$ helm install --devel --namespace ${RELEASE_NAMESPACE} hashicorp/terraform --generate-name
```

Comment:
This should probably just be ${NAMESPACE} to match usage above

Reply:
Done